packages/chartjs-plugin/playwright.config.ts

```diff
@@ -18,7 +18,8 @@ export default defineConfig({
   fullyParallel: true,
   forbidOnly: !!process.env.CI, // fail build on CI if we left test.only in the source code
   retries: process.env.CI ? 2 : 0, // retry on CI only
-  reporter: [["html", { open: "never" }]],
+  /* In the CI pipeline it generates github and blob reports, locally only a html report is generated */
+  reporter: process.env.CI ? [["github"], ["blob"]] : [["html", { open: "never" }]],
```

Comment:
the CI run is spammed with Playwright comments, is this caused by the github reporter?
<img width="296" alt="image" src="https://github.com/SchwarzIT/onyx/assets/67898185/bf4ffa95-1550-465e-ac68-cf6d3f7830e5">

Reply:
Yes, I replaced it with the dot reporter, when running in the CI now.

Reply:
I think you changed it for the headless package only, chartjs-plugin and sit-onyx still have the github reporter

Reply:
done